pkg/descheduler/strategies/duplicates.go

```diff
@@ -196,6 +201,8 @@ func RemoveDuplicatePods(
 						klog.ErrorS(err, "Error evicting pod", "pod", klog.KObj(pod))
 						break
 					}
+					metrics.DuplicatePods.With(map[string]string{"workloadName": ownerKey.name,
+						"workloadKind": ownerKey.kind, "namespace": pod.Namespace, "node": nodeName}).Dec()
```

Comment:
I'd like to see a test which will validate the new metric does not increase indefinitely. E.g. in case some change gets merged which will cause the metrics to only get increased.